docs/everest/backends/index.rst

```diff
@@ -8,8 +8,8 @@ Backends
     optimization
     sampling
 
-The optimization functionality of Everest is determined by optimization backends that provide the low-level optimization algorithms. For details about the optimization backends that are included with Everest, see section :ref:`cha_optimization_backends`.
+Ensemble-based optimization is handled internally by the `ropt <https://tno-ropt.github.io/ropt/>`_ library for robust optimization, which in turn uses a plugin mechanism to provide multiple low-level optimization backends. The different backends employ various well-tested optimization packages, providing the user with a wide choice of reliable algorithms, see section :ref:`cha_optimization_backends`.
```

Comment:
I would recommend line-breaking this long line  (and the other lines in this PR) for better compatibility with all kinds of editors.